internal/validation/validate_cluster.go

```diff
@@ -32,14 +32,14 @@ import (
 	"github.com/Azure/ARO-HCP/internal/api/arm"
 )
 
-func ValidateClusterCreate(ctx context.Context, newCluster *api.HCPOpenShiftCluster, validationPathMapper api.ValidationPathMapperFunc) field.ErrorList {
+func ValidateClusterCreate(ctx context.Context, subscription *arm.Subscription, newCluster *api.HCPOpenShiftCluster, validationPathMapper api.ValidationPathMapperFunc) field.ErrorList {
```

Comment:
this change isn't allowed.  Validation methods are static (only the new and old obj).  If non-static validation is needed, add it to the the admission package.  This separation makes it clear which pieces of validation are guaranteed (static) and which are suggestions likely to slip over time and become invalid (admission).

If we expect this option to go away in the future and be permanently on, it can be added to the Operation.Options.